ocs_ci/templates/provider-client-deployment/machineconfig_to_enable_nested_virtualization.yaml

```diff
@@ -0,0 +1,40 @@
+---
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: worker
+  name: 80-worker-enable-nested-virt
```

Comment:
do we need to enable nested virtualization for BareMetal machines? Can you pls check this? 
For me it seems we don't have "nested" virtualization, because the BMs are physical machines

Reply:
I am not sure about this, will change in future if needed.